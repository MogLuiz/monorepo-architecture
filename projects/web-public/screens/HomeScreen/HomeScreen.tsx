import { Text } from "@monorepo/design-system";
import { sum } from "@monorepo/utils/math/sum";

export function HomeScreen() {
  return (
    <div>
      <Text tag="h1">Home do Projeto</Text>
      <Text tag="p">@alura/utils/math/sum: sum(2,2) = {sum(2, 2)}</Text>
    </div>
  );
}
