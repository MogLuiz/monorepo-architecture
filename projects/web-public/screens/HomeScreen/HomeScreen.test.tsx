import { customRender } from "@monorepo/test-commons/react-testing-library";
import { HomeScreen } from "./HomeScreen";

const render = customRender();

describe("<HomeScreen />", () => {
  it("renders the page", () => {
    const { container } = render(<HomeScreen />);
    expect(container).toMatchSnapshot();
  });
});
