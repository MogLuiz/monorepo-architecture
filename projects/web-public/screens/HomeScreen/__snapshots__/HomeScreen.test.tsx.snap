// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders the page 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <div>
    <h1
      class="c0"
    >
      Home do Projeto
    </h1>
    <p
      class="c0"
    >
      @monorepo/utils/math/sum: sum(2,2) = 
      4
    </p>
  </div>
</div>
`;
